CONTRIBUTING.md

```diff
@@ -93,3 +93,81 @@ and making it executable with `chmod +x .git/hooks/pre-commit`.
 
 Please refer to the file `dummy/change.py` that contain a Dummy driver and
 some comments to help you get started.
+
+## Use docker commands for start Monocle
+
+The easiest way for developing Monocle project is to run Monocle without
```

Comment:
Not agree with that. I'm using compose to develop following that CONTRIBUTING guide and that's fine :)
Please remove those sentences.

Instead could you add a warning by telling that this process is for informational only. It is here for folks that don't have  or don't want to use docker-compose.